### ISSUE TITLE
archived project?

### ISSUE THREAD

**Issue by cxwx** [kaniaorzel/aws-code-whisperer.vim#2]

**Comment by kaniaorzel** [comment]
yes, I tried to reverse engineered VSCode plugin 1 year ago. But without success :( After some time appera [codeium](https://codeium.com/) (IMO way better and with support to vim). I forgot to archived project. Thanks @cxwx :)
